### ISSUE TITLE
Embed image in a note

### ISSUE THREAD

**Issue by notklaatu** [turtl/tracker#127]
It would be nice to be able to embed an image within a note. Example:

```
This is some text in a note.
p----------q
|  Image   |
b----------d
More text blah blah.
```

(This, without having to upload an image to a private server so you can hyperlink to it.)

Since it's possible to create an image as a note, maybe there's a way to reference that image from within a text note?

**Comment by orthecreedence** [comment]
Yeah, this is not currently possible without linking outside of the app to an external server. I think that linking between notes is a good way to handle this (#37).
